### PR TITLE
Inital Commit: it is not finished

### PR DESCRIPTION
**Pull request by verniIgn**

### CODE REVIEW

File: org_chart.js [hunk 1]
```diff
@@ -1,70 +1,43 @@
...
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
+Persons.prototype.setSkills = function(skill) {
+  for(let i = 0; i < this.skills.length; i++) {
+    if(this.skills[i] != skill) {
+      this.skills.push(skill)
+    }
```
Comment (carlosnsr) [inline org_chart.js:14]: Test out this function.  It's an infinite loop.

File: org_chart.js [hunk 2]
```diff
@@ -1,70 +1,43 @@
...
+}
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+let tim = new Persons ("Tim Berners-Lee", "tim.berners_lee@musky.com", "Computer Science","Networks" ); 
+let elon = new Persons ("Elon Musk", "elon.musk@musky.com", "Boss")
+let alan = new Persons ("Alan Turing", "alan.turing@musky.com", "Computer Scientist", "Cryptography")
+let grace = new Persons ("Grace Hopper", "grace.hopper@musky.com", "Computer Scientist", "Compilers")
+let donald  = new Persons ("Donald Knuth", "donald.knuth@musky.com", "Mathematician", "Algorithm Analysis")
```
Comment (carlosnsr) [inline org_chart.js:23]: WHat about the gender?

File: org_chart.js [hunk 3]
```diff
@@ -1,70 +1,43 @@
...
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
-}
-alan.skills.push("Mathematics")
+tim.setSkills("Mathematics")
+alan.setSkills("Cobol")
+grace.setSkills("Computational Complexity")
+donald.setSkills("Protocols")
```
Comment (carlosnsr) [inline org_chart.js:27]: `.setSkill` instead of `.setSkllis` as you only submitting only one.

File: org_chart.js [hunk 4]
```diff
@@ -1,70 +1,43 @@
...
+grace.manager = elon
+donald.manager = elon
 tim.manager = elon
 
-if (!tim.skills) {
-  tim.skills = []
-}
-tim.skills.push("Protocols")
-
 console.log(`${alan.name} is a ${alan.position} working on ${alan.project}.`)
```
Comment (carlosnsr) [inline org_chart.js:37]: These messages are another feature that could be DRY'd

File: org_chart.js [hunk 5]
```diff
@@ -1,70 +1,43 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+function Persons (name, email, position, project, gender) {
```
Comment (carlosnsr) [inline org_chart.js:1]: I wouldn't use a plural for the name, as this constructor creates ONE person object

File: org_chart.js [hunk 6]
```diff
@@ -1,70 +1,43 @@
...
-  email: "tim.berners_lee@musky.com",
-  position: "Computer Science",
-  project: "Networks"
-}
-let tim = new_hire
+// assign each a manager
+alan.manager = elon
+grace.manager = elon
+donald.manager = elon
 tim.manager = elon
```
Comment (carlosnsr) [inline org_chart.js:35]: This is another operation that should be on the Person object

**Review by carlosnsr — CHANGES_REQUESTED** [review]

**Comment by carlosnsr** [comment]
Please address the feedback and get back to me.

**Comment by verniIgn** [comment]
I did a PR with the wrong code. The code I showed you yesterday (the one
you saw) is not the one on this repo. I don't have the correct code anymore
because I had it on Repl.it.com and now had disappeared.

**Pull request closed without merging**